meson.build

```diff
@@ -9,7 +10,6 @@ lib = library(
   'cgif',
   'cgif.c',
   soversion : '0',
-  version : '0.0.1',
```

Comment:
This is probably still needed. Without this, the `libcgif.so.0.0.1` symlink will not be installed.